spiketoolkit/validation/quality_metrics.py

```diff
@@ -893,12 +954,22 @@ def compute_metrics(sorting, recording=None, sampling_frequency=None, isi_thresh
                                                       amp_frames_after=amp_frames_after,
                                                       recompute_info=recompute_info,
                                                       max_spikes_for_pca=max_spikes_for_pca,
+                                                      apply_filter=apply_filter, freq_min=freq_min, freq_max=freq_max,
                                                       save_features_props=save_features_props, seed=seed)
-    if 'snr' in metric_names:
+    elif 'amplitude_cutoff' in metric_names:
+        if recording is None:
+            raise ValueError("The recording cannot be None when computing amplitude cutoffs.")
+        else:
+            metric_calculator.compute_amplitudes(recording=recording, amp_method=amp_method, amp_peak=amp_peak,
+                                                 amp_frames_before=amp_frames_before,
+                                                 amp_frames_after=amp_frames_after, apply_filter=apply_filter, 
+                                                 freq_min=freq_min, freq_max=freq_max, 
+                                                 save_features_props=save_features_props, seed=seed)
+    elif 'snr' in metric_names:
```

Comment:
this logic will fail if one calls ['amplitude_cutoff', 'snr']. The last one should be if

Reply:
No, this should be fine. It will automatically set the recording in `compute_amplitudes`

Reply:
It is `elif` because each of the previous steps computes the next step